cmd/metrics-resource.go

```diff
@@ -361,29 +362,25 @@ func collectLocalResourceMetrics() {
 	collectDriveMetrics(m)
 }
 
-// populateInitialValues - populates the initial values
-// for drive stats and host uptime
-func populateInitialValues() {
-	initialDriveStatsMu.Lock()
+func initLatestValues() {
+	latestDriveStatsMu.Lock()
+	defer latestDriveStatsMu.Unlock()
```

Comment:
Can we make this lock narrower ? it is confusing to read it that way, or maybe collectLocalMetrics() needs it ?